Src/LogData.c

```diff
@@ -216,13 +218,35 @@ void logDataTask(void const* arg)
 
     if (f_mount(&fatfs, "SD:", 1) == FR_OK)
     {
-        HAL_GPIO_WritePin(LED1_GPIO_Port, LED1_Pin, 1);
-
-        if (f_open(&file, "SD:VanderAvionics.csv", FA_OPEN_APPEND | FA_READ | FA_WRITE) == FR_OK)
+        if (f_open(&file, "SD:VanderAvionics.csv", FA_OPEN_EXISTING) == FR_NO_FILE)
         {
```

Comment:
Add line HAL_GPIO_WritePin(LED1_GPIO_Port, LED1_Pin, 1) here. We want LED1 to turn on when we create a file.